kava-stats.js

```diff
@@ -391,19 +394,9 @@ const mapCdpParams = async (denoms, cdpParamsData) => {
       if(cdpParam) { limit = cdpParam.debtLimit }
     }
 
-    coins[denom] = { debtLimit: limit }
+    coins[denom] = { debtLimit: limit, stabilityFeePercentage: mappedStabilityFees[denom] }
```

Comment:
we're already looping through the denoms object here. Instead of doing it again below, we can reuse this and just add our piece of data to the object